package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ios-sim",
-  "version": "8.0.1",
+  "version": "8.0.2-dev",
```

Comment:
Is this change required?

If `-dev` is required, I think this should have been done after the `8.0.1` patch release so that people who submit PRs do not need to manage this.

I actually already merged in a PR into the `8.x` branch which released in `8.0.2` but the `package.json` still has `8.0.1` from that PR.

Reply:
I just followed what I saw in other commits on https://github.com/ios-control/ios-sim/commits/master I guess.